docs/src/app/components/pages/components/Card/Page.jsx

```diff
@@ -44,6 +47,13 @@ const CardPage = () => (
     >
       <CardExampleWithoutAvatar />
     </CodeExample>
+    <CodeExample
+      title="Controlled card"
```

Comment:
Could you make this "Controlled example" please?

It's inconsistent in some of the alphabetically earlier examples, but as we progressed through the docs we settled on that rather than calling out the component name in the example titles.

You could perhaps make the one above "Exxpandable example" while you're there :smile: